### DISCUSSION TITLE
关于2.5的周本图

### DISCUSSION THREAD

**Discussion by waiwaiex** [Arondight/Adachi-BOT#674] [General]
最近逛米游社看到大佬已经把雷神的周本更新进去了[https://upload-bbs.mihoyo.com/upload/2022/02/18/75833613/15b472dcd67a67016ece772e8528faf0_2513941823560578118.png](https://upload-bbs.mihoyo.com/upload/2022/02/18/75833613/15b472dcd67a67016ece772e8528faf0_2513941823560578118.png)

**Comment by mark9804** [comment]
感谢提醒，稍晚些时候就添加

**Comment by mark9804** [comment]
已经添加完成了，麻烦拉取一下代码并且重启机器人
